src/app/screens/LNURLAuth/index.tsx

```diff
@@ -50,19 +51,18 @@ function LNURLAuth() {
 
       if (response.success) {
         setSuccessMessage("Authenticated successfully.");
+        // ATTENTION: if this LNURL is called through `webln.lnurl` then we immediately return and return the response. This closes the window which means the user will NOT see the above successAction.
+        // We assume this is OK when it is called through webln.
+        if (navState.isPrompt) {
+          msg.reply(response);
+        }
       } else {
-        throw new Error("Auth status is not ok");
-      }
-
-      // ATTENTION: if this LNURL is called through `webln.lnurl` then we immediately return and return the response. This closes the window which means the user will NOT see the above successAction.
-      // We assume this is OK when it is called through webln.
-      if (navState.isPrompt) {
-        msg.reply(response);
+        toast.error(`Error: Auth status is not ok`);
       }
     } catch (e) {
       console.error(e);
       if (e instanceof Error) {
-        toast.error(`Error: ${e.message}`);
+        setErrorMessage(`Error: ${e.message}`);
```

Comment:
above we're using toast and here we set an error message. should those be in sync?

Reply:
Yes, changed it. I think currently we use the toast if i.e. a call fails. For other errors we can use the error-message.